### PR TITLE
data: nabadev

### PR DESCRIPTION
**Pull request by nabadev**
Added my data on linkfree

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5924"></a>

### CODE REVIEW

File: data/nabadev.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Nabajyoti Bharali",
+  "type": "personal",
+  "bio": "Self-taught web developer",
+  "avatar": "https://github.com/nabadev.png/",
```
Comment (khushalarora) [inline data/nabadev.json:5]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @nabadev, thank you for contributing :tada:

File: data/nabadev.json (hunk 1 above)
Comment (nabadev) [reply data/nabadev.json:5]: Thanks for the information :)

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks! :tada:

**Pull request merged**